flowcraft/generator/templates/bandage.nf

```diff
@@ -1,50 +1,36 @@
-if (params.reference{{param_id}} != null) {
-    process bandage_{{pid}} {
-        {% include "post.txt" ignore missing %}
-
-        tag { sample_id }
-        publishDir "reports/assembly/bandage_{{pid}}/$sample_id"
-
-        input:
-        set sample_id, file(assembly) from {{input_channel}}
-        file reference from Channel.fromPath(params.reference{{param_id}})
-
-        output:
-        file "*.png"
-        file "*.svg"
-        {% with task_name="bandage" %}
-        {%- include "compiler_channels.txt" ignore missing -%}
-        {% endwith %}
-
-        script:
+has_gfa1_{{pid}} = binding.hasVariable('gfa1_{{pid}}')
```

Comment:
Even though this is perfectly fine, this pattern is unusual in nextflow pipeline scripts. For future reference, I suggest adding a comment line explaining what this is doing, for instance:
```suggestion
// Boolean that informs if a GFA secondary channel has been connected 
// to this component, that is, if the channel has been created in the pipeline.
has_gfa1_{{pid}} = binding.hasVariable('gfa1_{{pid}}')
```

Reply:
Done.